configs/openmrs/initializer_config/concepts/disposition.csv

```diff
@@ -1,9 +1,8 @@
-Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Answers,Members,_version:1,_order:500
-efaf4399-601c-4479-aba2-21591a6eccae,,org.openmrs.module.emrapi:TRANSFER,Transfer Patient,Transfer Patient,Transfer Patient,Transfer Patient,,,Misc,N/A,,,,
-59e51158-44ec-45d1-8d8a-b5c31f2b3212,,org.openmrs.module.emrapi:ADMIT,Admit Patient,Admit Patient,Admit Patient,Admit Patient,,,Misc,N/A,,,,
-89cd4d9f-579a-4ee5-9353-f62704688ea0,,org.openmrs.module.emrapi:DISCHARGE,Discharge Patient,Discharge Patient,Discharge Patient,Discharge Patient,,,Misc,N/A,,,,
-b1d4bb65-1637-496c-8e11-05da30a2a1cb,,org.openmrs.module.emrapi:UNDO_DISCHARGE,Undo Discharge,Undo Discharge,Undo Discharge,Undo Discharge,,,Misc,N/A,,,,
-d664e628-2957-4b02-8a25-28a8cb10fee5,,org.openmrs.module.emrapi:Disposition,Disposition,Disposition,Disposition,Disposition,,,Question,Coded,Transfer Patient;Admit Patient;Discharge Patient;Undo Discharge,,,
-a6ea10ed-ede7-43e2-83e8-c6be5ca26d76,,org.openmrs.module.emrapi:DispositionNote,Disposition Note,Disposition Note,Disposition Note,Disposition Note,,,Misc,Text,,,,
-caa44025-9e30-4feb-8d2a-c58c91f36bfc,,org.openmrs.module.emrapi:Disposition Concept Set,Disposition Set,Disposition Set,Disposition Set,Disposition Set,,,Misc,N/A,,Disposition;Disposition Note,,
-c238289b-8533-485e-8d9e-665f6f01f210,,,Adt Notes,Adt Notes,Adt Notes,Adt Notes,,,Misc,Text,,,,
\ No newline at end of file
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Answers,Members,_version:1,_order:500
+77eafb3f-58d3-4397-a6dc-d2c06e9062f3,,CIEL:168619,Admit to hospital,Admit to hospital,,Misc,N/A,,,,
```

Comment:
Admit to hospital -> Hospitaliser
Admission Location (text/code) -> Unité d'admission (texte/code)
Internal Transfer Location (text/code) -> Unité de transfert interne (texte/code)
Internal facility transfer/referral -> Transférer
Discharge from hospital -> Terminer l'hospitalisation
Inpatient patient disposition -> Orientation du patient
Inpatient disposition construct -> Ensemble d'orientation du patient

Reply:
Please have a look
![Screenshot 2025-02-25 at 1 08 14 PM](https://github.com/user-attachments/assets/00e33cf4-ee79-4726-8097-f2300f3494ea)

Reply:
OK thanks.
Seems, like we're missing some more strings on this form. (Ward Admission, Ward Request and Ward details).

Let's translate to:
- Ward Admission -> Demande d'admission
- Ward Request -> Demande
- Ward details -> Information de service

Additionally, let's rename in the concepts:
- Admitted to location (text/code) -> Service/unité où admettre (texte/code)
- Internal transfer location (text/code) -> Service/unité où transférer (texte/code)

Reply:
![Screenshot 2025-02-26 at 1 06 16 PM](https://github.com/user-attachments/assets/69e33e01-673c-4e5b-8fca-9ad7c05c297a)